src/app/config/config.default.ts

```diff
@@ -2,7 +2,7 @@ import { Config } from './config.model';
 
 export const defaultConfig: Config = {
   octoprint: {
-    url: 'http://localhost:5000/api/',
+    url: 'http://localhost:3000',
```

Comment:
Why are you changing the default here? The /api/ is still required and port 3000 is not the default port.